types/voting.ts

```diff
@@ -1,7 +1,7 @@
 import { Null, bool, u64, Enum, Struct, Vec, Tuple, Option, u128 } from '@polkadot/types';
 import U8aFixed from '@polkadot/types/codec/U8aFixed';
+import { AccountId } from '@polkadot/types/interfaces';
```

Comment:
If we're using the string `'AccountId'` do we still need to import this?

Reply:
Yes.  Since this appears a bit later on -

```js
  get initiator (): AccountId {
    return this.get('initiator') as AccountId;
  }
```